packages/twenty-front/src/generated-metadata/graphql.ts

```diff
@@ -504,6 +505,7 @@ export type CreateApiKeyDto = {
   expiresAt: Scalars['String'];
   name: Scalars['String'];
   revokedAt?: InputMaybe<Scalars['String']>;
+  roleId: Scalars['String'];
```

Comment:
FYI in future PR type it as uuid (merging like this but @abdulrahmancodes is doing a codebase-wide fix PR now)